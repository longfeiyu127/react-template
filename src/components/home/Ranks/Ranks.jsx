import React from 'react';

const HomeRanks = () => <h2>排行榜2222</h2>;

export default HomeRanks;
