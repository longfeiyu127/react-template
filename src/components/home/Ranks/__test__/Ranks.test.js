import React from 'react';
import renderer from 'react-test-renderer';
import Ranks from '../Ranks';

describe('<Ranks />', () => {
  it('component render', () => {
    let ranks = renderer.create(<Ranks />).toJSON();
    expect(ranks).toMatchSnapshot();

    ranks = renderer.create(<Ranks winner="ç™½" />).toJSON();
    expect(ranks).toMatchSnapshot();
  });
});
