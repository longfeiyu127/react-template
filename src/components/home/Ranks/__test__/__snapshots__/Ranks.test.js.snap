// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ranks /> component render 1`] = `
<h2>
  排行榜22223333
</h2>
`;

exports[`<Ranks /> component render 2`] = `
<h2>
  排行榜22223333
</h2>
`;
